stubs/s2clientprotocol/METADATA.toml

```
@@ -0,0 +1,2 @@
+version = "5.*"
+requires = ["protobuf<4", "types-protobuf"]
```

`protobuf` isn't `py.typed` (hence why we have stubs for it in typeshed!), so I don't think having it as a dependency _as well as_ `types-protobuf` gets us anything here? Removing it should also fix the `stub_uploader` errors in CI:

```suggestion
requires = ["types-protobuf"]
```